Week7/app/src/main/java/com/kulloveth/covid19virustracker/data/db/NewsEntity.kt

```diff
@@ -0,0 +1,21 @@
+package com.kulloveth.covid19virustracker.data.db
+
+import android.os.Parcelable
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * News Entity with table name news
+ * for accessing the news update
+ * */
+@Entity(tableName = "news")
```

Comment:
As mentioned these classes can be broken into multiple classes to simplify and minimize the numerous tags but this is not counting against your score.

Reply:
if you mean separating room entity from api model, I think I actually did that as with this project I have NewsEntity for working with room and News for working with api same as the other endpoint